src/routes/index.svelte

```diff
@@ -184,6 +184,13 @@
         </p>
       </div>
     </div>
+    <a
+        href="https://leapdao.org/validate/"
```

Comment:
```suggestion
        href="/validate"
```
We can use relative link here :)